apps/project/tests/e2e_create_project_tile_map_service_test.py

```diff
@@ -357,3 +365,66 @@ def _test_tile_map_service(self, projectKey: str, filename: str):
         assert sanitized_tasks_actual == sanitized_tasks_expected, (
             "Differences found between expected and actual tasks in firebase."
         )
+
+        # Test push tasks results to firebase
+        contributor_user_groups = ContributorUserGroupFactory.create_batch(2, **self.user_resource_kwargs)
+        input_data = test_data["results_firebase_input_data"]
+
+        # Change project_fb_id to real firebase id
+        input_data[project_fb_id] = input_data.pop("project_firebase_id")
+
+        user_groups_input = {str(ug.firebase_id): True for ug in contributor_user_groups}
+
+        for group_key, group_value in input_data[project_fb_id].items():  # noqa: B007
+            contributor_keys = list(group_value.keys())
+
+            for old_key in contributor_keys:
+                new_key = self.contributor_user.firebase_id
+
+                # Change group_firebase_id to real firebase id
+                group_value[new_key] = group_value.pop(old_key)
+
+                # Replace userGroups
+                group_value[new_key]["userGroups"] = user_groups_input
+
+        ref_results = Config.FIREBASE_HELPER.ref(Config.FirebaseKeys.results_projects())
+        ref_results.set(input_data)
+        fb_results_data = ref_results.get()
+        assert fb_results_data is not None
+
+        # Check for empty data before data pull from firebase
+        assert [
+            MappingSession.objects.count(),
+            MappingSessionResult.objects.count(),
+            MappingSessionUserGroup.objects.count(),
+            MappingSessionUserGroupTemp.objects.count(),
+            MappingSessionResultTemp.objects.count(),
+        ] == [0, 0, 0, 0, 0]
+
+        pull_results_from_firebase()
+
+        # Check if data is pulled
+        assert [
+            MappingSession.objects.count(),
+            MappingSessionResult.objects.count(),
+            MappingSessionUserGroup.objects.count(),
+            MappingSessionUserGroupTemp.objects.count(),
+            MappingSessionResultTemp.objects.count(),
+        ] != [0, 0, 0, 0, 0]
+
+        mapping_session = MappingSession.objects.first()
+        assert mapping_session is not None
+        assert mapping_session.start_time is not None
+        assert mapping_session.end_time is not None
+
+        pull_firebase_data = {
+            "project_firebase_id": mapping_session.project_task_group.project.firebase_id,
+            "app_version": mapping_session.app_version,
+            "start_time": mapping_session.start_time.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z",
+            "end_time": mapping_session.end_time.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z",
+            "results_count": MappingSessionResult.objects.count(),
```

Comment:
Add other checks too!

```suggestion
             "mapping_session_count": MappingSession.objects.count(),
            "results_count": MappingSessionResult.objects.count(),
            "user_groups_count": MappingSessionUserGroup.objects.count(),

```